### ISSUE TITLE
Link role can be replaced by sound

### ISSUE THREAD

**Issue by cary-rowen** [nvaccess/nvda#16769]
### Is your feature request related to a problem? Please describe.
In browse mode, if multiple words in the text are marked as links, NVDA frequently reports "link" when announcing the text. This repetition can significantly disrupt the user's comprehension of the original content.

Consider the following example:

"When an link object that allows link navigation and/or editing of text is link focused, you can move through the text using the link system caret, also known as the link edit cursor."

However, link roles are crucial in this context, and users might miss important information if NVDA doesn't announce link roles.

### Describe the solution you'd like
Change the checkboxes under "link" in "Document formatting" to dropdowns, allowing users to select between off, speech, sound, speech and sound.

p.s. Use cases for Braille users should also be considered here.

Unlike line indentation (the tone of beep can be used to indicate the depth of indentation), for link we can use a more pleasant audio sound.

### Describe alternatives you've considered
None

### Additional context
An [add-on][withSounds] has already been developed to address this issue and is very popular within the NVDA Chinese community.

Additionally, TalkBack includes this feature as well.

[withSounds]: https://dl.nvdacn.com/Share/withSounds-0.0.2.nvda-addon

**Comment by hamidrezaabroshan** [comment]
Jaws also has such feature and links, headings and other html elements can be represented with sound.

**Comment by zh-yx** [comment]
That add-on was written by me. I think it makes sense to replace some of the formatting information and role types with sound effects.


The following information can be prompted using sound effects:

1. Link,Visited and Unvisited Link.
2. List item, Entering and exiting the list.
3. Clickable element.

The benefits of using sound effects as a substitute are as follows:

1. Separate original text from formatting information.
2. It doesn't sound too tiring

**Comment by josephsl** [comment]
Hi,

I believe there was a GitHub issue on NVDA sound schemes, including bringing parts of now unmaintained Audio Themes add-on and friends to the core. If yes, after reviewing this and similar issues, I vote to keep a more general issue open with referneces to specific contexts added.

Thanks.

**Comment by cary-rowen** [comment]
Hi @josephsl 

It would be great if anyone would be willing to triage these issues, I did a cursory search before creating this request and couldn't find any examples that closely matched this request.
cc @Adriani90 Can you find an existing feature request that closely matches this request and completely covers it?



Thanks

**Comment by Adriani90** [comment]
Related to #9738 and #4089.

**Comment by seanbudd** [comment]
Closing in favour of #9738
